splunk_topology/test_splunk_topology.py

```diff
@@ -207,7 +208,7 @@ def _mocked_finalize_sid_exception(*args, **kwargs):
         except CheckException:
             thrown = True
 
-        self.assertTrue(thrown)
+        self.assertFalse(thrown)
```

Comment:
do we now have tests that at least cover both cases, i.e. a case where `ignore_saved_search_errors` is set to `true` and a case where it set to `false`?

Reply:
added a new case for true.